online.php

```diff
@@ -169,7 +171,9 @@
 			break;
 		case "pgsql":
 		default:
-			$query = $db->simple_select("sessions", "COUNT(sid) as online", "time > {$timesearch}");
+			$query = $db->simple_select("sessions", "COUNT(sid) as online", "time > {$timesearch}", array(
+			    'group_by' => 'uid, ip',
+            ));
```

Comment:
Indentation here is off - it seems to use spaces for the closing `));` rather than tabs.

```suggestion
			));
```